### ISSUE TITLE
[Feature] [Deeprun Tram] Weird Behavior

### ISSUE THREAD

**Issue by Soge** [FeenixServerProject/Archangel_2.4.3_Bugtracker#19]
The deeprun tram movement starts to the opposite side and than bounces back to the correct side. If you are in the "first" node (the one closest to your destination) you will be moved back to the last-node place and hit the wall, staying at same city you were. It's REALLY hard to explain, better someone go see it for yourself.
3 "nodes" (i.e seats) leave Stormwind and only 2 arrive to ironforge (the object itself, is missing).
Basically, the entire train is messed up!

Also noticed that only 1 train is active instead of 2. the other train just moves out of the map.

special thanks goes to Malcret for pointing out on this issue
